pinot-broker/src/main/java/org/apache/pinot/broker/requesthandler/MultiStageBrokerRequestHandler.java

```diff
@@ -387,68 +388,8 @@ private void checkAuthorization(RequesterIdentity requesterIdentity, RequestCont
   private ImmutableQueryEnvironment.Config getQueryEnvConf(HttpHeaders httpHeaders, Map<String, String> queryOptions,
       long requestId) {
     String database = DatabaseUtils.extractDatabaseFromQueryRequest(queryOptions, httpHeaders);
-    boolean inferPartitionHint = _config.getProperty(CommonConstants.Broker.CONFIG_OF_INFER_PARTITION_HINT,
```

Comment:
This is moved to pinot-query-planner/src/main/java/org/apache/pinot/query/QueryEnvironmentConfigFactory.java